frontend/widgets/OBSBasic_SceneItems.cpp

```diff
@@ -35,6 +35,22 @@
 
 using namespace std;
 
+namespace {
+bool isHideInMixer(obs_source_t *source)
```

Comment:
```suggestion
bool isHiddenInMixer(obs_source_t *source)
```
Use natural language.